.yamato/package.metafile

```diff
@@ -0,0 +1,58 @@
+test_editors:
+  - version: trunk
+  - version: 2023.1
+  - version: 2022.2
+  - version: 2021.3
+  - version: 2020.3
+  - version: 2019.4
+
+promotion_test_editors:
+  - version: trunk
+  - version: 2019.4
+
+test_platforms:
+  - name: win
+    type: Unity::VM
+    image: package-ci/win10:v4
```

Comment:
This is a change from the previous package-ci/win10:stable and package-ci/macos-12:stable images which is a good thing as stable shouldn't be used anymore. I see the templates are using MacOS 13 rather than 12, do you think we should upgrade?

Reply:
Hmm, good question. I think in terms of ensuring compatibility with minimum requirements for 2019.4 we should be aiming for as low a MacOS version as possible as 2019.4 supports 10.12 and up. However I think MacOS 12 is the earliest we can actually support because of certain graphics test issues, in which case maybe using the older doesn't really matter as it's not the min spec anyway...